composer.json

```diff
@@ -8,14 +8,15 @@
         "php": "^7.3 || ^8.0",
         "silverstripe/recipe-plugin": "^1",
         "silverstripe/recipe-core": "^4",
-        "cwp/cwp-core": "^2",
-        "silverstripe/auditor": "^2",
-        "silverstripe/environmentcheck": "^2",
-        "silverstripe/hybridsessions": "^2"
+        "cwp/cwp-core": "^2.9",
+        "silverstripe/auditor": "^2.4",
+        "silverstripe/environmentcheck": "^2.4",
+        "silverstripe/hybridsessions": "^2.4"
```

Comment:
Why have these been increased?

Reply:
Fix prefer-lowest build so that we get phpunit9 compatible versions

phpcs file is not added because there is no php code to lint